{"uuid": "6f68462c-4642-4473-ba4a-38d83b22ff13", "children": ["44cb605d-025d-4d3e-a2cd-85b264c8b641"], "befores": [{"name": "set_maverick_mode_all", "status": "broken", "statusDetails": {"message": "UnboundLocalError: local variable 'iwinfo' referenced before assignment\n", "trace": "  File \"/home/imgd/TIP/venv/lib/python3.8/site-packages/pluggy/callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/imgd/TIP/venv/lib/python3.8/site-packages/_pytest/fixtures.py\", line 1126, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/home/imgd/TIP/venv/lib/python3.8/site-packages/_pytest/fixtures.py\", line 925, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/home/imgd/TIP/wlan-testing/tests/e2e/interOp/conftest.py\", line 644, in set_maverick_mode_all\n    print(\"iiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiii\",iwinfo)\n"}, "start": 1640787864303, "stop": 1640787875523}], "start": 1640787864303, "stop": 1640787875622}